modules/file/api.go

```diff
@@ -0,0 +1,8 @@
+package file
+
+import "io"
+
+// GetReadCloser returns an io.ReadCloser if the file name is valid.
```

Comment:
You should specify what that the `io.ReadCloser` is the file specified by fileName.